.travis.yml

```diff
@@ -20,6 +20,7 @@ matrix:
       env: TOXENV=py36
     - python: 3.7
       env: TOXENV=py37
+before_install: .travis/before_install.sh
```

Comment:
Per our convo at the CI meeting, please add a comment here on how you generated the other files, so that we can do so again in the future.